### PR TITLE
Added Systemdplugin

### PR DESCRIPTION
**Pull request by amno1**
Added functions for making it easier to work with systemd.

### CODE REVIEW

File: aliases/available/systemd.aliases.bash [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cite 'about-alias'
```
Comment (nwinkler) [inline aliases/available/systemd.aliases.bash]: I think I have already merged this in #1014...

File: plugins/available/systemd.plugin.bash [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Author: Arthur Miller
+# Created: 2017-08-08
+# License: Public Domain
+
+cite about-plugin
+about-plugin 'SystemD helper functions'
+
+# Reloads a systemwide service. If no agruments are given, reloads all services.
```
Comment (nwinkler) [inline plugins/available/systemd.plugin.bash]: Typo in `agruments`, should be `arguments`...

File: plugins/available/systemd.plugin.bash [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+# Requires admin priviledge.
+scrs(){
```
Comment (nwinkler) [inline plugins/available/systemd.plugin.bash]: A couple of general comments for all of these functions:

1. The name should be descriptive, don't use an abbreviation here. Use something like `systemd_reload_service`. If you want a shorter version of that, add a shorter alias in the `systemd.aliases.bash` after that.
1. Please add the meta tags for each function, these are used for describing the available functions. Please see https://github.com/Bash-it/bash-it/blob/master/plugins/available/base.plugin.bash#L21 for an example of what to add for each function.

File: plugins/available/systemd.plugin.bash [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    if [ $# -eq 0 ]; then
+	systemctl daemon-reload
```
Comment (nwinkler) [inline plugins/available/systemd.plugin.bash]: The indentation seems to be off for these lines. Can you please indent the lines with 2 spaces per level? The same goes for the other functions - they all seem to be off...

File: plugins/available/systemd.plugin.bash [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+# status for all services. Requires admin priviledge.
+scst(){
+    if [ $# -eq 0 ]; then
+	systemctl status
+    else
+	systemctl status "$1.service"
+    fi;
+}
+
+# Reloads an user service. If no agruments are given, reloads all user services.
```
Comment (nwinkler) [inline plugins/available/systemd.plugin.bash]: Two typos here:

1. It should be `Reloads a user service.`, not `an user service.`
1. `agruments` --> `arguments`

**Comment by NoahGorny** [comment]
Hey, seems like this PR has gone stale. So I am closing it. Thanks for contributing!

**Pull request closed without merging**